thrift/src/main/thrift/atoms/storyquestions.thrift

```diff
@@ -0,0 +1,27 @@
+namespace * contentatom.storyquestions
+namespace java com.gu.contentatom.thrift.atom.storyquestions
+#@namespace scala com.gu.contentatom.thrift.atom.storyquestions
+
+enum RelatedStoryLinkType {
+  TAG = 0,
+  STORY = 1,
+}
+
+struct Question {
```

Comment:
Yes, but not yet. Plus this will need more thought - an answer could be an atom (explainer) or a piece of content etc so I'm keeping it simple for now.